hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/hevm/ActionSidecarContentTracer.java

```diff
@@ -51,4 +51,9 @@ public interface ActionSidecarContentTracer extends OperationTracer {
      * @param type the type of precompile called; expected values are {@code PRECOMPILE} and {@code SYSTEM}
      */
     void tracePrecompileResult(@NonNull MessageFrame frame, @NonNull ContractActionType type);
+
+    /**
+     * TODO: Add documentation
+     */
```

Comment:
Since I'm not sure about the implementation - I will add the documentation when the code is finalized.